pages/instructorAssessmentManualGrading/instructorAssessmentManualGrading.ejs

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <%- include('../partials/head'); %>
+    <link href="<%= node_modules_asset_path('tablesorter/dist/css/theme.bootstrap.min.css') %>" rel="stylesheet" />
+    <script src="<%= node_modules_asset_path('tablesorter/dist/js/jquery.tablesorter.min.js') %>"></script>
+    <script src="<%= node_modules_asset_path('tablesorter/dist/js/jquery.tablesorter.widgets.min.js') %>"></script>
+  </head>
+  <body>
+    <script>
+      $(function () {
+          $('[data-toggle="popover"]').popover({sanitize: false})
+      });
+    </script>
+    <%- include('../partials/navbar'); %>
+    <div id="content" class="container-fluid">
+      <%- include('../partials/assessmentSyncErrorsAndWarnings'); %>
+
+      <!-- ###################################################################### -->
+      <!-- ###################################################################### -->
+
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white">
+          <%= assessment_set.name %> <%= assessment.number %>: Manual Grading Queue
+        </div>
+
+        <div class="card-body">
+          <p class="small">
+            A list of ungraded submissions for questions. Note that submissions that students have saved, but not submitted, may be listed here.
```

Comment:
I think this note would be better in the card-footer (see the Access page). Also,
```suggestion
            This list may include answers that students saved but did not request to be graded.
```

Reply:
My concern with this living in the footer is that there may be a large number of student questions, so the notice may be buried.

However, I'll move it into the footer since it seems like all of the other instructor pages like to put things in the footer.